package.json

```diff
@@ -16,6 +18,7 @@
 		"url": "git+https://github.com/bitfocus/companion-module-generic-websocket.git"
 	},
 	"devDependencies": {
-		"@companion-module/tools": "^1.0.2"
+		"@companion-module/tools": "^1.0.2",
+		"prettier": "^2.8.7"
```

Comment:
Prettier is already installed as part of `@companion-module/tools`, you don't need to install it explicitly. But there is no harm in doing so